pytket/tests/requirements.txt

```diff
@@ -6,5 +6,5 @@ pytest-benchmark
 py
 hypothesis
 docker
-jsonschema
+jsonschema==4.18.0
```

Comment:
Is this change pushed on purpose?

Reply:
Yes, we need this, 4.18.1 seems to have a bug that causes test failures :-(